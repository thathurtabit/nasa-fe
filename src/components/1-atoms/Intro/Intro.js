import React from 'react';
import IntroStyled from './Intro.styled';
import { splitSpanString } from '../../../utils/helpers/splitSpanString';

const Intro = () => (
  <IntroStyled>Celebrate a {splitSpanString('new job')}</IntroStyled>
);

export default Intro;
