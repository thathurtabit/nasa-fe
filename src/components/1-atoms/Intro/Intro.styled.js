import styled from 'styled-components';

const IntroStyled = styled.h1`
  color: ${props => props.theme.colors.copy};
  font-size: calc(${props => props.theme.fonts.baseSize} + 1vmin);
  text-transform: lowercase;

  span {
    color: ${props => props.theme.colors.title};

    &:nth-child(even) {
      position: relative;
      top: -2px;
    }
  }
`;

export default IntroStyled;
