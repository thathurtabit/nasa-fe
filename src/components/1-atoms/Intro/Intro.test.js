import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import 'jest-styled-components';
import { shallowWithTheme } from '../../../utils/test_config/testHelpers';

import Intro from './Intro';

describe('Intro', () => {
  it('should render correctly', () => {
    const element = shallowWithTheme(<Intro />);
    expect(shallowToJson(element)).toMatchSnapshot();
  });
});
