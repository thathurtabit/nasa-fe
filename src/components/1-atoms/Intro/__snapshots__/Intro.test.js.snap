// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
